client/src/GlobalStyle.js

```diff
@@ -12,14 +12,25 @@ html {
   font-family: "Orbitron", Arial, sans-serif;
   background-image: url(${img});
   background-repeat: no-repeat;
-  background-size: 100vw;
+  background-size: 100vmax;
+  display: flex;
+  justify-content: center;
 
```

Comment:
hey there, that should be build into the body not in the html area

Reply:
Thanks 👍🏻